### PR TITLE
client: fix doc typo

### PR DESCRIPTION
**Pull request by hnlq715**
fix doc typo

### CODE REVIEW

File: client/v3/naming/doc.go [hunk 1]
```diff
@@ -39,7 +39,7 @@
 //	func etcdDial(c *clientv3.Client, service string) (*grpc.ClientConn, error) {
 //		etcdResolver, err := resolver.NewBuilder(c);
 //		if err { return nil, err }
-//		return  grpc.Dial("etc://foo", grpc.WithResolvers(etcdResolver))
+//		return  grpc.Dial("etcd://foo", grpc.WithResolvers(etcdResolver))
```
Comment (ptabor) [inline client/v3/naming/doc.go:42]: Please make it: etcd:///foo

Please see:  `tests/integration/clientv3/naming/resolver_test.go`  as working example.

**Review by ptabor — CHANGES_REQUESTED** [review]
Thank you for fixing it.

File: client/v3/naming/doc.go (hunk 1 above)
Comment (hnlq715) [reply client/v3/naming/doc.go]: done :-)

**Comment by ptabor** [comment]
Thank you.

**Pull request merged**